crates/types/Cargo.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "rjacraft_types"
```

Comment:
Very ambiguous naming. Could we change the name something along the lines of `rjacraft_mc_data`? Just `rjacraft_data`? @JarvisCraft